### PR TITLE
Genesis: add FW versions for G70 2020 Korean edition

### PR DESCRIPTION
**Pull request by chesterroh**

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Which route did these versions come from?

**Comment by chesterroh** [comment]
Genesis 2020 G70 which was sold in Korea domestic market. 
G70 3.3T AWD purchased in 2020 Apr ( 2020 Model Year )

The car is mainly being driven in metropolitan Seoul area.

**Comment by chesterroh** [comment]
> Which route did these versions come from?

[b8f31ec96ab82e13|2022-05-15--22-22-50](https://useradmin.comma.ai/?onebox=b8f31ec96ab82e13%7C2022-05-15--22-22-50)

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -805,21 +805,27 @@ class Buttons:
...
       b'\xf1\x00IK  MDPS R 1.00 1.07 57700-G9220 4I2VL107',
       b'\xf1\x00IK  MDPS R 1.00 1.07 57700-G9420 4I4VL107',
+      b'\xf1\x00IK  MDPS R 1.00 1.08 57700-G9420 4I4VL108',
     ],
     (Ecu.transmission, 0x7e1, None): [
       b'\xf1\x87VCJLP18407832DN3\x88vXfvUVT\x97eFU\x87d7v\x88eVeveFU\x89\x98\x7f\xff\xb2\xb0\xf1\x81E25\x00\x00\x00',
       b'\x00\x00\x00\x00\xf1\x00bcsh8p54  E25\x00\x00\x00\x00\x00\x00\x00SIK0T33NB4\xecE\xefL',
       b'\xf1\x87VDKLT18912362DN4wfVfwefeveVUwfvw\x88vWfvUFU\x89\xa9\x8f\xff\x87w\xf1\x81E25\x00\x00\x00\x00\x00\x00\x00\xf1\x00bcsh8p54  E25\x00\x00\x00\x00\x00\x00\x00SIK0T33NB4\xecE\xefL',
+      b'\xf1\x87VDJLC18480772DK9\x88eHfwfff\x87eFUeDEU\x98eFe\x86T5DVyo\xff\x87s\xf1\x81E25\x00\x00\x00\x00\x00\x00\x00\xf1\x00bcsh8p54  E25\x00\x00\x00\x00\x00\x00\x00SIK0T33KB5\x9f\xa5&\x81',
+
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:815]:
```suggestion
      b'\xf1\x87VDJLC18480772DK9\x88eHfwfff\x87eFUeDEU\x98eFe\x86T5DVyo\xff\x87s\xf1\x81E25\x00\x00\x00\x00\x00\x00\x00\xf1\x00bcsh8p54  E25\x00\x00\x00\x00\x00\x00\x00SIK0T33KB5\x9f\xa5&\x81',
```

**Comment by sshane** [comment]
Make sure to keep your device connected to WiFi to upload logs

**Comment by sshane** [comment]
Thanks for the PR! Looks good

**Pull request merged**